README.md

```diff
@@ -456,17 +456,25 @@ Uses [Jackson](https://github.com/FasterXML/jackson) to look for the records in
            Implementation based on based on a `DateTimeFormatter`
 >     *   `com.github.castorm.kafka.connect.http.response.timestamp.NattyTimestampParser`
            Implementation based on [Natty](http://natty.joestelmach.com/) parser
+>     *   `com.github.castorm.kafka.connect.http.response.timestamp.RegexTimestampParser`
+           Implementation that tries to extract substring timestamp from the field and parse into correct format 
 > 
 > ##### `http.response.record.timestamp.parser.pattern`
 > When using `DateTimeFormatterTimestampParser`, a custom pattern can be specified 
 > *   Type: `String`
-> *   Default: `yyyy-MM-dd'T'HH:mm:ss[.SSS]X`
+> *   Default: `yyyy-MM-dd'T'HH:mm:ss[.SSS]X` 
+> *   Special: `L` in case `http.response.record.timestamp.parser = com.github.castorm.kafka.connect.http.response.timestamp.RegexTimestampParser` and the substring is in milli seconds
```

Comment:
May I suggest to remove this "special" case? I provide more details in the code.

Reply:
Yes I have removed the special case and added a delegate to support further formatting